internal/cmd/update.go

```diff
@@ -0,0 +1,108 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/sonatype-nexus-community/nancy/buildversion"
+	"github.com/sonatype-nexus-community/nancy/update"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(updateCmd)
+}
+
+var updateCmd = newUpdateCommand()
+
+func newUpdateCommand() *cobra.Command {
+
+	updateCmd := &cobra.Command{
+		Use: "update",
+		//Short: "Update the tool to the latest version",
+		Short: "Check if there are any updates available",
+		RunE: func(_ *cobra.Command, _ []string) error {
+			//return updateCLI("", true)
+			return updateCLI("", false)
+		},
+	}
+
+	/*	updateCmd.AddCommand(&cobra.Command{
+			Use:    "check",
+			Hidden: true,
+			Short:  "Check if there are any updates available",
+			RunE: func(_ *cobra.Command, _ []string) error {
+				return updateCLI("", false)
+			},
+		})
+	*/
+	/*	updateCmd.AddCommand(&cobra.Command{
+			Use:    "install",
+			Hidden: true,
+			Short:  "Update the tool to the latest version",
+			PersistentPreRun: func(_ *cobra.Command, _ []string) {
+				opts.cfg.SkipUpdateCheck = true
+			},
+			PreRun: func(cmd *cobra.Command, args []string) {
+				opts.args = args
+			},
+			RunE: func(_ *cobra.Command, _ []string) error {
+				return updateCLI(opts)
+			},
+		})
+	*/
+	//updateCmd.PersistentFlags().BoolVar(&opts.dryRun, "check", false, "Check if there are any updates available without installing")
+
+	return updateCmd
+}
+
+func logAndShowMessage(message string) {
+	logLady.Info(message)
+	fmt.Println(message)
```

Comment:
Do we want to respect the quiet flag here??

Reply:
Maybe its fine.... realistically this is for humans only right?? Not really a CI thing and probably doesn't hurt to log some more.

Reply:
I see your thinking, and am still leaning to leaving the "nag" enabled even in "quiet" mode. If somebody complains, we can tweak things.